ext/pgsql/pgsql.stub.php

```diff
@@ -947,6 +947,8 @@ function pg_set_error_context_visibility(PgSql\Connection $connection, int $visi
 #ifdef HAVE_PG_RESULT_MEMORY_SIZE
     function pg_result_memory_size(PgSql\Result $result): int {}
 #endif
+
+    function pg_change_password(PgSql\Connection $connection, string $user, string $password): bool {}
```

Comment:
The password parameter should have the SensitiveParameter attribute

Reply:
ah very true I forgot about this.